tests/src/packages/samples/GreetingTests.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package packages.samples;
+
+import static common.Pair.make;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.google.code.tempusfugit.concurrency.ParallelRunner;
```

Comment:
@rabbah I like to keep this PR as simple as possible. Why remove ParallelRunner? Since there is too few test cases in this PR, ParallelRunner does not take too much effect or no effect. Besides, the naming convention is very suspicious. Why do we use com.google.code.tempusfugit.concurrency as the package name?? It is not a openwhisk package name. I do not hope it is part of some other existing code in other projects.